elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/ElasticsearchSourceSettings.scala

```diff
@@ -8,10 +8,14 @@ package akka.stream.alpakka.elasticsearch
  * Configure Elastiscsearch sources.
  *
  */
-final class ElasticsearchSourceSettings private (val bufferSize: Int, val includeDocumentVersion: Boolean) {
+final class ElasticsearchSourceSettings private (val bufferSize: Int,
+                                                 val includeDocumentVersion: Boolean,
+                                                 val scroll: String) {
```

Comment:
Maybe it would be preferable to restrict to `FiniteDuration` for `scroll`. But we'd need conversions to the appropriate `String` as described in the [Elasticsearch docs](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/common-options.html#time-units).
The least you could do is to point to the ES docs from a doc comment.

Reply:
Thank you for the review.
Changes done: [bfcd7e4](https://github.com/akka/alpakka/pull/1366/commits/bfcd7e42ba954bad55d25543ea531dfad715a2b5)